payload/Library/installapplications/installapplications.py

```diff
@@ -610,6 +610,18 @@ def main():
                 continue
             iaslog('%s processing %s %s at %s' % (stage, type, name, path))
 
+            # On userland stage, we want to wait until we are actually
+            # in the user's session.
+            if stage == 'userland':
+                if len(iajson['userland']) > 0:
+                    while (getconsoleuser()[0] is None
+                           or getconsoleuser()[0] == u'loginwindow'
```

Comment:
don't think we need the unicode force on the string, considering it wasn't needed in my new `postinstall`

Reply:
Missed that during rebase. Thanks. -> fcd2778